firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -87,10 +87,11 @@ def get_split_chars_base(entry):
         if "⚑" in entry.custom_order or has_ignored_char:
             return []
         else:
-            # convert title to base characters
-            base_title = alphabet.get_base_form(entry.title)
+            # split, then convert title to base characters
```

Comment:
I changed this back to split, then convert to base as it does come up in edge cases. For example if "c" "h" and "ch" are all characters with specific variants.